versions/draft-3/SPEC.md

```diff
@@ -247,6 +247,8 @@ Float f = 27.3             # A floating point number
 Boolean b = true           # A boolean true/false
 String s = "hello, world"  # A string value
 File f = "/path/to/file"   # A file
+SFile u1 = "gs://bucket/file" # URI for Google cloud storage file
```

Comment:
I don't like `SFile` as the datatype.

I think that `DataObject` might be better.